src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -86,55 +85,44 @@ public class SendReceiveTest : ClientWebSocketTestBase
         {
             using (ClientWebSocket cws = await WebSocketHelper.GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
             {
-                var cts = new CancellationTokenSource(TimeOutMilliseconds);
+                var tasks = new Task[2];
 
-                Task[] tasks = new Task[10];
+                // Need large buffer, or Send completes too quickly and no exception is thrown.
+                var sendBuffer = new byte[128 * 1024];
+                var segment = new ArraySegment<byte>(sendBuffer);
```

Comment:
The buffer will be all zeros, yet it's being sent as Text.  It'd be better to have it filled will actual UTF8 text, as it was previously.